<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="THIYAGUVASU" failures="0" tests="4" name="com.AdactineMavenProject.DependsOnMethod" time="0.020" errors="1" timestamp="2022-06-20T23:14:44 IST" skipped="0">
  <testcase classname="com.AdactineMavenProject.DependsOnMethod" name="Vistara" time="0.003"/>
  <system-out/>
  <testcase classname="com.AdactineMavenProject.DependsOnMethod" name="indiGo" time="0.002"/>
  <system-out/>
  <testcase classname="com.AdactineMavenProject.DependsOnMethod" name="flightTicketsOffer" time="0.013"/>
  <system-out/>
  <testcase classname="com.AdactineMavenProject.DependsOnMethod" name="spicejet" time="0.002">
    <error message="
Method DependsOnMethod.spicejet()[pri:0, instance:com.AdactineMavenProject.DependsOnMethod@55536d9e] should have thrown an exception of type class org.testng.TestNG" type="org.testng.TestException">
      <![CDATA[org.testng.TestException: 
Method DependsOnMethod.spicejet()[pri:0, instance:com.AdactineMavenProject.DependsOnMethod@55536d9e] should have thrown an exception of type class org.testng.TestNG
at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:82)
at org.testng.internal.TestInvoker.considerExceptions(TestInvoker.java:749)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:634)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- spicejet -->
  <system-out/>
</testsuite> <!-- com.AdactineMavenProject.DependsOnMethod -->
